src/generators/operationGenerator.ts

```diff
@@ -198,6 +199,67 @@ function buildMapper(
   return `${mapperName}: ${mapperString},`;
 }
 
+/**
+ * This function gets the parameter groups specified in the swagger
+ * by the parameter grouping extension x-ms-parameter-grouping
+ */
+function getGroupedParameters(
+  operation: OperationDetails,
+  parameters: ParameterDetails[],
+  importedModels: Set<string>
+): ParameterWithDescription[] {
+  let parameterGroups: Parameter[] = [];
```

Comment:
Nit: prefer const over let
```suggestion
  const parameterGroups: Parameter[] = [];
```